packages/test/local-server-stress-tests/src/test/localServerStressOrderSequentially.spec.ts

```diff
@@ -0,0 +1,97 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { done, isOperationType, takeAsync } from "@fluid-private/stochastic-test-utils";
+
+import { makeGenerator, reducer, saveFailures, type StressOperations } from "../baseModel.js";
+import {
+	convertToRealHandles,
+	covertLocalServerStateToDdsState,
+	DDSModelOpGenerator,
+	loadAllHandles,
+	validateConsistencyOfAllDDS,
+	type DDSModelOp,
+	type OrderSequentially,
+} from "../ddsOperations.js";
+import {
+	createLocalServerStressSuite,
+	LocalServerStressModel,
+	type LocalServerStressState,
+} from "../localServerStressHarness.js";
+
+const orderSequentiallyReducer = async (
+	state: LocalServerStressState,
+	op: OrderSequentially,
+) => {
+	const { baseModel, taggedHandles } = await loadAllHandles(state);
+	const ddsState = await covertLocalServerStateToDdsState(state);
+	const rollbackError = new Error("rollback");
+	try {
+		state.datastore.orderSequentially(() => {
+			for (const o of op.operations) {
+				baseModel.reducer(ddsState, convertToRealHandles(o, taggedHandles));
+				if (op.rollback) {
+					throw rollbackError;
+				}
+			}
+		});
+	} catch (error) {
+		if (error !== rollbackError) {
+			throw error;
+		}
+	}
+};
+
+describe("Local Server Stress with rollback", () => {
+	const model: LocalServerStressModel<StressOperations | OrderSequentially> = {
+		workloadName: "rollback",
+		generatorFactory: () =>
+			takeAsync(
+				100,
+				makeGenerator<OrderSequentially>([
+					[
+						async (state) => {
+							const operations: DDSModelOp[] = [];
+							/**
+							 * unfortunately we can't generate more than a single op here, as each op is generated off
+							 * the current state, and if we generate multiple ops it can result in earlier ops invaliding
+							 * the constrains necessary for later ops. for example, an earlier op might delete a sub-directory
+							 * which a later op sets a key in, but the state and generator don't know that will happen.
+							 */
+							const op = await DDSModelOpGenerator(state);
+							if (op !== done) {
+								operations.push(op);
+							}
+							return {
+								type: "orderSequentially",
+								operations,
+								rollback: true,
+							} satisfies OrderSequentially;
+						},
+						50,
+					],
+				]),
+			),
+		reducer: async (state, op) =>
+			isOperationType<OrderSequentially>("orderSequentially", op)
+				? orderSequentiallyReducer(state, op)
+				: reducer(state, op),
+		validateConsistency: validateConsistencyOfAllDDS,
+	};
+
+	createLocalServerStressSuite(model, {
+		defaultTestCount: 100,
+		// skipMinimization: true,
+		// Uncomment to replay a particular seed.
+		// only: [91],
+		saveFailures,
+		// saveSuccesses,
+		configurations: { "Fluid.ContainerRuntime.EnableRollback": true },
+		skip: [
+			3, 4, 9, 10, 13, 16, 21, 23, 27, 28, 33, 35, 37, 38, 39, 40, 47, 48, 49, 52, 56, 63, 68,
```

Comment:
ouch!

Reply:
meh. its like 3 different errors, and we have bug to fix them. just want the test in first.